src/clj/game/cards/events.clj

```diff
@@ -242,6 +242,17 @@
                                    (effect-completed state side eid)))))}
              :run-ends {:effect (effect (unregister-events card))}}}
 
+   "Credit Kiting"
+   {:req (req (some #{:hq :rd :archives} (:successful-run runner-reg)))
+    :prompt "Select a card to install from your Grip"
+    :choices {:req #(and (or (is-type? % "Hardware")
+                             (is-type? % "Program")
+                             (is-type? % "Resource"))
+                         (in-hand? %))}
+    :effect (effect (install-cost-bonus [:credit -8])
+                    (runner-install target)
+                    (gain :tag 1))}
```

Comment:
I believe the tag should be preventable as it's not a cost. Instead of `gain`, you should call `tag-runner`.